vid/src/advz.rs

```diff
@@ -741,12 +762,12 @@ where
     fn evaluate_polys(
         &self,
         polys: &[DensePolynomial<<E as Pairing>::ScalarField>],
+        code_word_size: usize,
```

Comment:
I wonder whether this arg should be `multiplicity` instead of `code_word_size`. Then we refactor the multiplication  `num_storage_nodes * multiplicity` inside this function instead of relying on the caller to do it each time. Also, at first glance it seems the purpose of most/all use of `code_word_size` outside this function is to prepare an arg for this function or `bytes_to_polys` or `polynomial`. In that case it's a no-brainier.

Reply:
Yes, I had initially just passed in `multiplicity` as argument to the functions but changed it again since IMO fx
`let all_storage_node_evals = self.evaluate_polys(&polys, code_word_size as usize)?;` provides slightly better readability than just passing in the `multiplicity`.